lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -514,12 +514,26 @@ function dosomething_reportback_form_alter(&$form, &$form_state, $form_id) {
 function dosomething_reportback_view_entity($entity) {
   $rb_simple_user = dosomething_user_get_simple_user($rb_user = user_load($entity->uid));
   global $user;
+
   $is_owner = ($user->uid == $rb_simple_user->uid) ? 1 : 0;
   $fid = $_GET['fid'];
+
   if (!$fid) {
     // Get the most recently uploaded item.
     $fid = array_pop($entity->fids);
   }
+
+  $disable_reactions = dosomething_kudos_disable_reactions($entity->nid);
```

Comment:
Mind adding a `@TODO` for a future refactor run to DRY up some of this code. Seems like there's overlap between what is going on here and in the [preprocess_node.inc](https://github.com/DoSomething/phoenix/blob/280b2937a41c600e001916ba7b6dcd35fadc81a1/lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_node.inc#L253) and [helpers.inc](https://github.com/DoSomething/phoenix/blob/280b2937a41c600e001916ba7b6dcd35fadc81a1/lib/themes/dosomething/paraneue_dosomething/includes/helpers.inc#L354) etc... could be good to consolidate into a single function that preps that stuff if possible!